tests/acceptance/repo/trigger-build-test.js

```diff
@@ -0,0 +1,84 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'travis/tests/helpers/module-for-acceptance';
+import triggerBuildPage from 'travis/tests/pages/trigger-build';
+
+moduleForAcceptance('Acceptance | repo/trigger build', {
+  beforeEach() {
+
+  }
+});
```

Comment:
If there’s nothing in `beforeEach`, the entire second argument to `moduleForAcceptance` is safely left out.